IntegrationTests/tests_01_http/test_19_connection_drop_while_waiting_for_response_uds.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+##===----------------------------------------------------------------------===##
+##
+## This source file is part of the SwiftNIO open source project
+##
+## Copyright (c) 2017-2018 Apple Inc. and the SwiftNIO project authors
+## Licensed under Apache License v2.0
+##
+## See LICENSE.txt for license information
+## See CONTRIBUTORS.txt for the list of SwiftNIO project authors
+##
+## SPDX-License-Identifier: Apache-2.0
+##
+##===----------------------------------------------------------------------===##
+
+source defines.sh
+
+token=$(create_token)
+start_server --disable-half-closure "$token"
+htdocs=$(get_htdocs "$token")
+server_pid=$(get_server_pid "$token")
+socket=$(get_socket "$token")
+
+kill -0 $server_pid
+echo -e 'GET /dynamic/write-delay/10000 HTTP/1.1\r\n\r\n' | nc -w1 -U "$socket"
+sleep 0.2
+stop_server "$token"
```

Comment:
Does this test actually "fail" in any particular way?

Reply:
@Lukasa yes, in case it leaks fds or if it crashes. The old code wouldn't have passed this. I acknowledge this is not very obvious...

Reply:
@weissi maybe also add a comment explain this ;) ?

Reply:
@weissi please address this one ;)